### PR TITLE
Cart & wishlist pages

### PR DESCRIPTION
**Pull request by amartya-rs**
feat: wishlist and cart page
## Files to check

- [src/context/cart-context.js ] 
- [ src/components/cart-card/CartCard.jsx] 
- [src/components/wishlist-card/WishlistCard.jsx ] 
- [src/pages/cart-page/CartPage.jsx ] 
- [src/pages/wishlist-page/WishlistPage.jsx ] 

Screenshot (372)

### CODE REVIEW

File: src/components/cart-card/CartCard.jsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+   };
+
+   return (
+      <div className="card horizontal">
+         <img className="card-img" src={card.imgUrl} alt="product" />
+         <div className="card-content">
+            <h6 className="font-semibold">{card.title}</h6>
+            <div className="price-wrapper">
+               <h5>{`₹ ${card.discountPrice}`}</h5>
+               <h6 className="strike-through">{`₹ ${card.price}`}</h6>
```
Comment (rutu14) [inline src/components/cart-card/CartCard.jsx:15]: Suggestion: Instead of adding Rupee sign, can use the toLocaleString to get the currency symbol.

**Review by rutu14 — APPROVED** [review]
The structure looks good to merge.

**Review by dev-prabhat — CHANGES_REQUESTED** [review]
Code is clean and understandable, you can merge after working on the given suggestion

File: src/components/cart-card/CartCard.jsx (hunk 1 above)
Comment (amartya-rs) [reply src/components/cart-card/CartCard.jsx:15]: Will work on it. Thanks for your suggestion.

**Pull request merged**